src/commands/game/index.ts

```diff
@@ -222,23 +223,46 @@ async function runGame(options: GameOptions): Promise<GameResult | null> {
       winner,
       preText: pre
     }
+  } catch (ex) {
+    await postFailMessage(bot, channel, ex)
+    return null
+  }
+}
+
+async function tryGetSelection(
+  bot: SlackClient,
+  channel: string,
+  userId: string,
+  username: string,
+  mode: Mode,
+  timeout: number,
+  preText?: string
+) {
+  try {
+    const selection = await getSelection(bot, mode, userId, timeout, preText)
+    return selection
   } catch (ex) {
     if (ex instanceof TimeoutError) {
       await bot.postMessage({
         channel,
-        text: `Looks like ${challenger} and ${opponent} chickened out! :hatched_chick:`
+        text: `Looks like ${username} chickened out! :hatched_chick:`
```

Comment:
Both users could chicken out. Instead of throwing the TimeoutError, we can just return the error or some other sentinel value.
Something like:
```ts
if (!left || !right) {
  const chickens: string[] = []
  if (!left) { chickens.push(challenger) }
  if (!right) { chickens.push(opponent) }
  await bot.postMessage({ chicken out message })
  return null
}
```

Reply:
Hmm, yeah that looks a lot cleaner. Do you mean returning `null` instead of throwing in `getSelection(...)`?

Reply:
Yeah. Throwing was a good way to exit the loop, but your method of always returning a result will let us see if one or both players chicken out.
```ts
{ text: `It looks like ${chickens.join(' and ')} chickend out!`
```